src/test/java/org/apache/commons/beanutils2/converters/PeriodConverterTestCase.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.beanutils2.converters;
+
+
+import java.time.Period;
+
+import org.apache.commons.beanutils2.ConversionException;
+import org.apache.commons.beanutils2.Converter;
+
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+
+/**
+ * Test Case for the PeriodConverter class.
+ *
+ */
+public class PeriodConverterTestCase extends TestCase {
+
+    public static TestSuite suite() {
+        return new TestSuite(PeriodConverterTestCase.class);
+    }
+
+    // ------------------------------------------------------------------------
+
+    private Converter converter = null;
+
+    // ------------------------------------------------------------------------
+
+    public PeriodConverterTestCase(final String name) {
+        super(name);
+    }
+
+    protected Class<?> getExpectedType() {
+        return Period.class;
+    }
+
+    protected Converter makeConverter() {
+        return new PeriodConverter();
+    }
+
+    // ------------------------------------------------------------------------
+
+    @Override
+    public void setUp() throws Exception {
+        converter = makeConverter();
+    }
+
+    @Override
+    public void tearDown() throws Exception {
+        converter = null;
+    }
+
+    // ------------------------------------------------------------------------
+
+    public void testSimpleConversion() throws Exception {
+        final String[] message= {
+            "from String",
+            "from String",
+            "from String",
+            "from String",
+            "from String",
+            "from String",
+            "from String",
+            "from String",
+        };
+
+        final Object[] input = {
+        	"P2Y",
+        	"P5D",
+        	"P1Y2M3D"
+        };
+
+        final Period[] expected = {
+        		Period.parse("P2Y"),
+        		Period.parse("P5D"),
+        		Period.parse("P1Y2M3D")
+        };
+
+        for(int i=0;i<expected.length;i++) {
```

Comment:
You are missing spaces for formatting in the for loop. We can probably do one fat reformatting commit later and save IDE settings in git.